core/src/test/scala/unit/kafka/admin/ConsumerGroupCommandTest.scala

```diff
@@ -113,8 +114,8 @@ class ConsumerGroupCommandTest extends KafkaServerTestHarness {
 
 object ConsumerGroupCommandTest {
 
-  abstract class AbstractConsumerRunnable(broker: String, groupId: String) extends Runnable {
-    val props = new Properties
+  abstract class AbstractConsumerRunnable(broker: String, groupId: String, customPropsOpt: Option[Properties] = None) extends Runnable {
+    val props = if (customPropsOpt.isDefined) new Properties(customPropsOpt.get) else new Properties
```

Comment:
Wouldn't it be better to move this line after `configure(props)` below so we also provide a way to override the default configs? In other words, this line would remain unchanged, and after `configure(props)` we add
```
    if (customPropsOpt.isDefined)
      props.putAll(customPropsOpt.get)
```